docs/topics/Automated-Checks.md

```diff
@@ -0,0 +1,259 @@
+# Automated Checks
+
+The [](DataWorkerService.md) has numerous responsibilities, one of them being a data processing step known as automated
+checks. These checks are responsible for processing various portions of data depending on the current processing step
+for a particular piece of data.
```

Comment:
why the weird line breaks :ehW:

```suggestion
The [](DataWorkerService.md) has numerous responsibilities, one of them being a data processing step known as automated checks. These checks are responsible for processing various kinds of data and moving them through different processing steps.
```

Reply:
meh writerside was complaining I shouldn't have listened.